src/renderer/components/+nodes/nodes.tsx

```diff
@@ -98,74 +98,45 @@ export class Nodes extends React.Component<Props> {
     });
   }
 
-  renderCpuUsage(node: Node) {
-    const metrics = this.getLastMetricValues(node, ["cpuUsage", "cpuCapacity"]);
-
-    if (!metrics || !metrics[1]) return <LineProgress value={0}/>;
-    const usage = metrics[0];
-    const cores = metrics[1];
-    const cpuUsagePercent = Math.ceil(usage * 100) / cores;
-    const cpuUsagePercentLabel: String = cpuUsagePercent % 1 === 0
-      ? cpuUsagePercent.toString()
-      : cpuUsagePercent.toFixed(2);
+  renderUsage(node: Node, title: string, metricNames: [string, string], secondFormater: (metrics: [number, number]) => string) {
```

Comment:
IMHO when having lots of arguments, it's start to get painful to call those methods. If you don't believe, please try to use kubernetes client library 😄

Reply:
I switched it to a destructed object. I was mostly asking about the zero arguments. Because I think that functions should be as pure as possible to help reduce the mental load when reading the code and if a function has no arguments then it is almost certainly not pure.